apps/src/templates/studioHomepages/CourseBlocksTools.jsx

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import ContentContainer from '../ContentContainer';
+import ResourceCard from './ResourceCard';
+import styleConstants from '../../styleConstants';
+import i18n from "@cdo/locale";
+
+const pegasusContentWidth = styleConstants['content-width'];
+
+const styles = {
+  container: {
+    width: pegasusContentWidth,
+    display: "flex",
+    justifyContent: "space-between"
+  },
+  regularRow: {
+    marginBottom: 20
+  }
+};
+
+const CourseBlocksTools = React.createClass({
+  propTypes: {
+    isEnglish: React.PropTypes.bool.isRequired,
+    isRtl: React.PropTypes.bool.isRequired,
+    codeOrgUrlPrefix: React.PropTypes.string.isRequired
```

Comment:
Ditch `codeOrgUrlPrefix` and use [urlHelpers.js](https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/lib/util/urlHelpers.js) instead: 

```jsx
import {pegasus} from '@cdo/apps/lib/util/urlHelpers';
//...
<ResourceCard
  link={pegasus(`/${card.path}`)}
/>
```